ch08.adoc

```diff
@@ -432,112 +435,52 @@ Note that an implementation of the interpolation method is free to calculate the
-  float m_sol_azi_ang(tp_track, tp_scan) ;
-    m_sol_azi_ang : standard_name = "solar_azimuth_angle" ;
-    m_sol_azi_ang : units = "degrees" ;
-  float m_sol_zen_ang(tp_track, tp_scan) ;
-    m_sol_zen_ang : standard_name = "solar_zenith_angle" ;
-    m_sol_zen_ang : units = "degrees" ;
-
-  float i_lat(tp_track, tp_scan) ;
-    i_lat : standard_name = "latitude" ;
-    i_lat : units = "degrees_north" ;
-  float i_lon(tp_track, tp_scan) ;
-    i_lon : standard_name = "longitude" ;
-    i_lon : units = "degrees_east" ;
-  float i_sen_azi_ang(tp_track, tp_scan) ;
-    i_sen_azi_ang : standard_name = "sensor_azimuth_angle" ;
-    i_sen_azi_ang : units = "degrees" ;
-  float i_sen_zen_ang(tp_track, tp_scan) ;
-    i_sen_zen_ang : standard_name = "sensor_zenith_angle" ;
-    i_sen_zen_ang : units = "degrees" ;
-  float i_sol_azi_ang(tp_track, tp_scan) ;
-    i_sol_azi_ang : standard_name = "solar_azimuth_angle" ;
-    i_sol_azi_ang : units = "degrees" ;
-  float i_sol_zen_ang(tp_track, tp_scan) ;
-    i_sol_zen_ang : standard_name = "solar_zenith_angle" ;
-    i_sol_zen_ang : units = "degrees" ;
+  // VIIRS I-Band Channel 01 and 04
+  float I01_radiance(track, scan) ;
+    I01_radiance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I01_radiance:standard_name = "toa_outgoing_radiance_per_unit_wavelength" ;
+    I01_radiance:units = "W m-2 sr-1 m-1" ;
+  float I01_reflectance(track, scan) ;
+    I01_reflectance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I01_reflectance:long_name = "reflectance" ;
+    I01_reflectance:units = "1" ;
+
+  float I04_radiance(track, scan) ;
+    I04_radiance:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I04_radiance:standard_name = "toa_outgoing_radiance_per_unit_wavelength" ;
+    I04_radiance:units = "W m-2 sr-1 m-1" ;
+  float I04_brightness_temperature(track, scan) ;
+    I04_brightness_temperature:tie_points = "lat: lon: tp_interpolation  t: time_interpolation" ;
+    I04_brightness_temperature:standard_name = "brightness_temperature" ;
+    I04_brightness_temperature:units = "K" ;
 
   // Interpolation variable
   char tp_interpolation ;
-    tp_interpolation:interpolation_name = "bi_quadratic_1" ;
-    tp_interpolation:interpolation_parameters =
-         "exp1: expansion1
-          align1: alignment1
-          exp2: expansion2
-          align2:alignment2 
-          flags: interpolation_zone_flags" ;
-
-  // Interpolation coefficient and configuration variables
-  short expansion1(zone_track , tp_scan) ;
-  short alignment1(zone_track , tp_scan) ;
-  short expansion2(tp_track, zone_scan) ;
-  short alignment2(tp_track, zone_scan) ;
+    tp_interpolation:interpolation_name = "bi_quadratic_remote_sensing" ;
+    tp_interpolation:tie_point_dimensions = "track: tp_track zone_track
+                                             scan: tp_scan zone_scan
+                                             scan: tp_time_scan"  ;
```

Comment:
(Comment A)

```suggestion
;
```
`t` is the only tie point variable spanning the `tp_time_scan` dimension and there is no data variable in the CDL stating that `t` must be interpolated using `tp_interpolation` (they all use `time_interpolation`), so I am not sure if defining this mapping here is relevant.

Reply:
Valid point, thank you for the careful review :-)